appinventor/appengine/src/com/google/appinventor/client/TopToolbar.java

```diff
@@ -186,8 +186,11 @@ public void updateMenuState(int numSelectedProjects, int numProjects) {
     boolean allowDelete = !isReadOnly && numSelectedProjects > 0;
     boolean allowExport = numSelectedProjects > 0;
     boolean allowExportAll = numProjects > 0;
+    String exportProjectLabel = numSelectedProjects > 1 ?
+        MESSAGES.exportSelectedProjectsMenuItem() : MESSAGES.exportProjectMenuItem();
```

Comment:
It'd be nice if we change the message so that it takes the numeric argument so the string is something like "Export N selected projects"